sentinel/transitions/index.js

```diff
@@ -92,45 +65,51 @@ const processChange = (change, callback) => {
   if (change.deleted) {
     // don't run transitions on deleted docs, but do clean up
     async.parallel([
-      async.apply(deleteInfoDoc, change),
+      callback => {
+        infodoc.delete(change)
+          .then(() => { callback(); })
+          .catch(err => { callback(err); });
+      },
       async.apply(deleteReadDocs, change)
     ], err => {
       if (err) {
         logger.error('Error cleaning up deleted doc', err);
       }
       processed++;
-      updateMetaData(change.seq, callback);
+      metadata.update(change.seq)
+        .then(() => { callback(); })
+        .catch(() => { callback(); });
```

Comment:
This is throwing away the err. Shouldn't it be `.catch((err) => { callback(err); })` ?